src/data.js

```diff
@@ -26,10 +26,11 @@ var Client = Entity.configure({
     expires:        Entity.types.Date,
     /**
      * Details object with properties:
-     * - created          // Time when client was created
-     * - lastModified     // Last time client was modified
-     * - lastDateUsed     // Only updated if more than 6 hours out of date
-     * - lastRotated      // Last time accessToken was reset
+     * - created            // Time when client was created
+     * - lastModified       // Last time client was modified
+     * - lastDateUsed       // Only updated if more than 6 hours out of date
+     * - lastRotated        // Last time accessToken was reset
+     * - deleteOnExpiration // If true, can be deleted after expiration
```

Comment:
Either do a migrate function to next version... or declare that  `deleteOnExpiration` is optional.